manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -16,7 +16,8 @@ webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
       {{- if .Values.istiodRemote.injectionURL }}
-      url: {{ .Values.istiodRemote.injectionURL }}
+      # the url doesn't take query params, so use inject/cluster/x/net/y format
+      url: {{ .Values.istiodRemote.injectionURL }}/cluster/{{ .Values.global.multiCluster.clusterName | default "remote0" }}/net/{{ .Values.global.network | default "network0" }}
```

Comment:
IMO a better format is `inject/cluster=x;net=y` so we can extend it with `inject/template=foo` or some other option in the future as needed. But we can change in the future if needed

Reply:
I'm not a big fan of having `=` or `;` in the url when it is not query params.  It just feels so odd.  The nice thing is user doesn't need to configure this whole url.